webui/react/src/components/RemainingRetentionDaysLabelComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import { Loadable } from 'hew/utils/loadable';
+import React from 'react';
+
+import { pluralizer } from 'utils/string';
+
+import Badge from './Badge';
+
+interface Props {
+  remainingLogDays: Loadable<number | undefined>;
```

Comment:
nit: i think we should unwrap `Loadable` before passing the props.

Reply:
done